functions/tsconfig.json

```diff
@@ -1,35 +1,36 @@
 {
-  "compilerOptions": {
-    "module": "commonjs",
-    "noImplicitReturns": true,
-    "noUnusedLocals": true,
-    "outDir": "lib",
-    "sourceMap": true,
+    "compilerOptions": {
+        "paths": {
+            "#generated/*": ["./generated/*"],
+            "#components/*": ["./src/components/*"],
+            "#views/*": ["./src/views/*"],
+            "#hooks/*": ["./src/hooks/*"],
+            "#utils/*": ["./src/utils/*"],
+            "#resources/*": ["./src/resources/*"],
+            "#base/*": ["./src/Base/*"]
```

Comment:
Let's not update the tsconfig.json file. The paths are not correct for this project.